specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2019-08-01/cosmos-db.json

```diff
@@ -5201,6 +5969,65 @@
         }
       }
     },
+    "SqlStoredProcedureResource": {
+      "type": "object",
+      "description": "Cosmos DB SQL storedProcedure resource object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Name of the Cosmos DB SQL storedProcedure"
+        },
+        "body":{
+          "type": "string",
+          "description": "Body of the Stored Procedure"
+        }
+      },
+      "required": [
+        "id"
+      ]
+    },
+    "SqlUserDefinedFunctionResource": {
```

Comment:
Can you clone this for SqlStoredProcedure? Your descriptions won't make sense for the stored proc api.

Reply:
I didnt understand what the change you are expecting. Can you please elaborate.

Reply:
I'll put a comment in a better place to describe this.